packages/ui5-proxy-middleware/src/base/manifest.ts

```diff
@@ -0,0 +1,2953 @@
+/* tslint:disable */
```

Comment:
This file is generated, isn't it? Can you add a note somewhere how and when this is done?

Reply:
Yes, done with https://github.com/SAP/open-ux-tools/pull/249/commits/539d7ee28cce7d540fb01cfa0e5596486b100371